pkg/meta/userdb/userdb.go

```diff
@@ -0,0 +1,307 @@
+package userdb
```

Comment:
I think this package should be under pkg/meta/userdb/boltdb-wrapper/userdb.go, instead of directly udner userdb.